sql/firefox_accounts_derived/fxa_users_services_first_seen_v1/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+friendly_name: FxA Users Services First Seen
+description: Usage aggregations describing when each FxA user was first seen
+owners:
+  - jklukas@mozilla.com
+labels:
+  application: fxa
+  incremental: true
+  schedule: daily
+scheduling:
+  dag_name: bqetl_fxa_events
+  # This query updates the entire existing table every day rather than appending
+  # a new partition, so we need to disable date_partition_parameter and instead
+  # pass submission_date as a generic param.
+  date_partition_parameter: null
+  parameters: ["submission_date:DATE:{{ds}}"]
+  sql_file_path: sql/firefox_accounts_derived/fxa_users_services_first_seen_v1/init.sql
```

Comment:
I think we might need to add some extra support for having `destination_table` as `None`? https://github.com/mozilla/telemetry-airflow/blob/caa17a72b905adc8e975d9310814032855abea87/dags/fxa_events.py#L157

Reply:
Good catch. I will look into it.